covid-projections/covid-projections
# Fix Invalid DOM property warning for States SVG

Fixes the Invalid DOM property warning for SVG states.

In development, React is throwing a warning saying that `clip-rule` should be `clipRule` and that `fill-rule` should be `fillRule`. This PR fixes both properties for all the files in `src/components/StateSvg`, and nothing else.

![image](https://user-images.githubusercontent.com/114084/80070910-9e406e80-84f8-11ea-99db-d97583d68be3.png)

## Testing
- Run the web app locally
- Go to http://localhost:3000/us/md
- Confirm that the Invalid DOM property errors for `clip-rule` and `fill-rule` don't show up anymore
